.github/workflows/CI.yml

```diff
@@ -10,14 +10,18 @@ on:
         type: boolean
   pull_request:
   push:
+    branches:
+      - 'master'
+      - 'stable-*'
+    tags: '*'
   schedule:
     # Every day at 2:30 AM UTC
     - cron: '30 2 * * *'
 
 concurrency:
   # group by workflow and ref; the last slightly strange component ensures that for pull
-  # requests, we limit to 1 concurrent job, but for the master branch we don't
-  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.ref != 'refs/heads/master' || github.run_number }}
+  # requests, we limit to 1 concurrent job, but for the default repository branch we don't
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.ref_name != github.event.repository.default_branch || github.run_number }}
```

Comment:
This change is not really important for our repository, but when copy&pasting this rule from here it is useful to have it agnostic from the name of the default branch.